lib/web_ui/lib/platform_dispatcher.dart

```diff
@@ -96,7 +95,7 @@ abstract class PlatformDispatcher {
 
   Locale get locale;
 
-  List<Locale> get locales => configuration.locales;
+  List<Locale> get locales;
```

Comment:
Thanks for doing this, the indirection here with the `configuration` getter that was overridden in the "engine" platform dispatcher was very confusing!